storage/inmemory_storage.py

```diff
@@ -14,6 +16,19 @@ def add_element(self, service_key, feed_element):
         key = service_key + hashlib.md5(body.encode('utf-8')).hexdigest()
         if not key in self.dict:
             self.dict[key] = feed_element
+        self.data_save(key, feed_element)
 
     def find_elements(self, service_key, filter_lambda):
         return {k: v for k, v in self.dict.items() if k.startswith(service_key) and filter_lambda(v)}
+    
+    def data_save(self, key, feed_element):
```

Comment:
InMemory storage should exist in memory. I will be really grateful if you implement database storage, but please do it as a separate StorageBase inheritor.